rasa/shared/constants.py

```diff
@@ -72,3 +72,6 @@
 DEFAULT_NLU_RESULTS_PATH = "nlu_comparison_results"
 DEFAULT_CORE_SUBDIRECTORY_NAME = "core"
 DEFAULT_NLU_SUBDIRECTORY_NAME = "nlu"
+
+DIAGNOSTIC_DATA = "diagnostic_data"
+ATTENTION_WEIGHTS = "attention_weights"
```

Comment:
these constants should not be in shared package as they are ML model specific.

Reply:
Ok. This is true for `ATTENTION_WEIGHTS`, but I thought of `DIAGNOSTIC_DATA` as something that can be returned by any model (most just don't). Don't have a strong opinion on that, though.